### PR TITLE
[ISSUE #475] add unit test depend mysql

### LINKED ISSUE #475: 恢复service中的单元测试 & 添加docker-mysql作为数据源进行集成测试
**What is the feature you want to add?**

**Why do you want to add this feature?**

**How to implement this feature?**

**Additional context**

### PR DESCRIPTION
**Pull request by chuntaojun**
**Please provide issue(s) of this PR:**
Fixes #475 

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [x] Naming
- [ ] HealthCheck
- [x] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: service/batch/future.go [hunk 1]
```diff
@@ -119,3 +118,14 @@ func sendReply(futures interface{}, code uint32, result error) {
 		log.Errorf("[Controller] not found reply futures type: %T", futures)
 	}
 }
+
+func reportRegisInstanceCost(begin, cur time.Time, code uint32) {
+	if code != api.ExecuteSuccess {
```
Comment (shichaoyuan) [inline service/batch/future.go:123]: 这里是否需要把失败的耗时也统计进来？

Comment (chuntaojun) [reply service/batch/future.go:123]: 失败就不计算时间了，只统计有效的，失败的话，只关注失败原因了

**Review by andrewshan — APPROVED** [review]
LGTM

**Review by daheige — APPROVED** [review]
LGTM

**Pull request merged**